django/contrib/humanize/templatetags/humanize.py

```diff
@@ -219,9 +219,8 @@ def naturaltime(value):
     if value < now:
         delta = now - value
         if delta.days != 0:
-            return pgettext(
-                'naturaltime', '%(delta)s ago'
-            ) % {'delta': defaultfilters.timesince(value, now)}
+            # Translators: delta will contain a string like '2 months' or '1 month, 2 weeks'
+            return _('%(delta)s ago') % {'delta': defaultfilters.timesince(value, now)}
```

Comment:
Quick sanity check for me: this changes `pgettext` for `gettext`, that's intended right?

Reply:
Thanks.